src/styles/app.css

```diff
@@ -1,134 +1,312 @@
   font-style: normal;
   line-height: normal;
   letter-spacing: -0.2px;
   color: #121212;
+  display: block;
 }
 
-.Suzanne-Collins {
+.author {
   width: 6.688rem;
   height: 1.188rem;
   margin: 0.25rem 0 0;
+  font-family: RobotoSlab;
   font-size: 0.875rem;
   font-weight: 300;
   font-stretch: normal;
   font-style: normal;
   line-height: normal;
   letter-spacing: normal;
   color: #4386bf;
+  display: block;
 }
 
-button {
+.but {
+  width: 4.563rem;
+  height: 1.188rem;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+  font-family: RobotoSlab;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
   line-height: normal;
   letter-spacing: normal;
   color: #4386bf;
-  background-color: transparent;
-  border: none;
-  cursor: pointer;
 }
 
-.Comments {
-  width: 4.563rem;
-  height: 1.188rem;
-  margin: 0.188rem 0.938rem 0.125rem 0;
+.line {
+  width: 0.125rem;
+  height: 1.5rem;
+  margin: 0.75rem 1.063rem 0 0.938rem;
+  border: solid 1px #e8e8e8;
+  color: #e8e8e8;
+}
+
+.oval-2 {
+  width: 4.25rem;
+  height: 4.25rem;
+}
+
+.encad-2 {
+  display: flex;
+}
+
+.percent {
+  width: 2.938rem;
+  height: 2.438rem;
+  margin: 0.063rem 7.063rem 0 0.688rem;
+  font-family: Montserrat;
+  font-size: 2rem;
+  color: #121212;
+  display: block;
+}
+
+.complete {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0.438rem 4.938rem 0.75rem 0.75rem;
+  opacity: 0.5;
+  font-family: Montserrat;
   font-size: 0.875rem;
+  color: #121212;
+}
+
+.line-big {
+  width: 0.125rem;
+  height: 4.375rem;
+  margin: 1.125rem 3.688rem 0 4.938rem;
+  border: solid 1px var(--white);
+}
+
+.current {
+  width: 7.875rem;
+  height: 1.125rem;
+  margin: 0.063rem 5.375rem 0 3.688rem;
+  opacity: 0.5;
+  font-family: RobotoSlab;
+  font-size: 0.813rem;
   font-weight: 300;
   font-stretch: normal;
   font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  display: block;
 }
 
-.Remove {
-  width: 3.375rem;
-  height: 1.188rem;
-  margin: 0.188rem 0.938rem 0.125rem 0;
-  font-size: 0.875rem;
+.current-lesson {
+  width: 13.25rem;
+  height: 1.313rem;
+  margin: 0.438rem 0 0.25rem 3.688rem;
+  font-family: RobotoSlab;
+  font-size: 1rem;
   font-weight: 300;
   font-stretch: normal;
   font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
 }
 
-.Edit {
-  width: 1.688rem;
-  height: 1.188rem;
-  margin: 0.188rem 18.625rem 0.125rem 1.063rem;
-  font-size: 0.875rem;
+.rectangle {
+  width: 11.5rem;
+  height: 2.063rem;
+  margin: 0.75rem 1.75rem 0.563rem 3.688rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+}
+
+.update {
+  width: 8.938rem;
+  height: 1.125rem;
+  font-family: RobotoSlab;
+  font-size: 0.813rem;
   font-weight: 300;
   font-stretch: normal;
   font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
 }
 
-.Line {
+.line-long {
```

Comment:
- [ ] Kindly add the horizontal line marked in red in the image below
- [ ] Also, kindly ensure your form section follows the spacing of the [given design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)

### Given design
<img width="1218" alt="Screenshot 2023-06-16 at 14 52 27" src="https://github.com/francksefu/booksmr/assets/64774936/2e361544-cdc5-4a80-9e36-9077b3178b29">

### Your design
<img width="1359" alt="Screenshot 2023-06-16 at 14 47 18" src="https://github.com/francksefu/booksmr/assets/64774936/519053c6-ce07-4905-b13f-ae94526cd12d">